doc/en/reference/reference.rst

```diff
@@ -1815,17 +1815,18 @@ All the command-line flags can be obtained by running ``pytest --help``::
                             expression. An expression is a python evaluatable
                             expression where all names are substring-matched
                             against test names and their parent classes.
-                            Example: -k 'test_method or test_other' matches all
+                            Example: -k "test_method or test_other" matches all
                             test functions and classes whose name contains
-                            'test_method' or 'test_other', while -k 'not
-                            test_method' matches those that don't contain
-                            'test_method' in their names. -k 'not test_method
-                            and not test_other' will eliminate the matches.
+                            "test_method" or "test_other", while -k "not
```

Comment:
I thought we wanted to use `'` in general, with `"` (per `cmd.exe`) being the exception?

Reply:
Oh btw, you don't need to update this text, it is generated automatically when we build the documentation.

Reply:
I think there might some confusion here but @taigit request changes in this [reply](https://github.com/pytest-dev/pytest/issues/9431#issuecomment-1001625011)

Reply:
Not sure, but regardless this file is updated automatically so any changes will get overwritten anyway.

Reply:
Right, changes to this file should be dropped. This means it wouldn't have helped @taigit but I don't think there is enough space in the pytest `--help` to explain the issue, and for most users `'` is the better quoting.